.github/workflows/ci.yml

```diff
@@ -174,7 +174,7 @@ jobs:
         uses: actions/upload-artifact@v3
         with:
           name: nginx-agent-snapshots
-          path: ./build/packages/nginx-agent.tar.gz
+          path: ./build/azure/packages/
```

Comment:
I think this should be `./build/azure/packages/nginx-agent.tar.gz`

Reply:
I'll change it there, thought it needed to be the path of what was being uploaded